examples/zephyr_example/zephyr_sample.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * ESP Flasher Library Example for Zephyr
+ * Written in 2022 by KT-Elektronik, Klaucke und Partner GmbH
+ * This example code is in the Public Domain (or CC0 licensed, at your option.)
+ * Unless required by applicable law or agreed to in writing, this
+ * software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
```

Comment:
I know I asked you a sample code before, but this one cannot be build using Zephyr's environment as is. I have created a proper sample code for that and to enable CI testing. Would you remove this `zephyr_example` folder from this PR? I'll submit the proper sample and will keep this copyright.